src/Listener/ApiListener.php

```diff
@@ -151,7 +152,8 @@ protected function _checkRequestMethods()
      */
     public function registerExceptionHandler()
     {
-        Configure::write('Error.exceptionRenderer', 'Crud\Error\ExceptionRenderer');
+        $config = ['exceptionRenderer' => 'Crud\Error\ExceptionRenderer'];
```

Comment:
How about something like this:

```php
(new ErrorHandler($config + (array)Configure::read('Error')))->register();
```

People would be required to change the `Configure::consume()` in app to `Configure::read()` until a better solution is found there? As it is, it's broken or requires more work than just this change to one's bootstrap if using the `ApiListener`. What do you think?